.examples/dockerfiles/full/fpm/Dockerfile

```diff
@@ -4,19 +4,18 @@ RUN mkdir -p /usr/share/man/man1 \
     && apt-get update && apt-get install -y \
         supervisor \
         ffmpeg \
-        libmagickwand-dev \
         libgmp3-dev \
         libc-client-dev \
         libkrb5-dev \
         smbclient \
         libsmbclient-dev \
-#       libreoffice \
```

Comment:
please keep this line. it's referenced from the readme:
```md
NOTE: The Dockerfile does not install the LibreOffice package (line is commented), because it would increase the generated Image size by approximately 500 MB. In order to install it, simply uncomment the 13th line of the Dockerfile.
```

Reply:
Good catch, I figured someone just forgot to delete the line. I've added it back. Line number was kept the same, as well. It looked like maybe the comment would break the escaped line return, but I guess blank lines in `RUN` commands don't get passed to the shell? I recall putting comments in the middle of escaped lines breaks the continuity of the shell command when working with normal bash scripts.